cli/test/utils/ds.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path');
+const child_process = require('child_process');
+
+module.exports = ds;
+
+const binPath = path.resolve(__dirname, '..', '..', 'lib', 'main.js');
+
+const defaultOptions = { stdio: 'pipe' };
+
+/**
+ * Helper method that allows to run ds command as if it
+ * is used from the command line
+ *  
+ * @param {string | string[]} args Argumets passed to the forked process 
+ * @param {ForkOptions} options Fork options
+ * 
+ * @returns {Promise<{ stdout: string, stderr: string, code: number }>}
+ */
+function ds(args, options) {
+  return new Promise((resolve, reject) => {
+    const subprocess = child_process.fork(
```

Comment:
Maybe have a look at [`execa`](https://github.com/sindresorhus/execa) and [`execa.command`](https://github.com/sindresorhus/execa#execacommandcommand-options) in particular.

Reply:
`execa.command()` is not released yet, but we are working hard on it. But yes, this would be great if `ds` used `execa` under the hood. It'd make the code simpler (e.g. this file would not be needed), while benefiting from nice things like better Windows support. https://github.com/sindresorhus/execa#why